index.html

```diff
@@ -5,111 +5,106 @@
+				<span class="pricing-table__price-sub">Per tour</span>
 			</div>
-
-			<div class="text"> <!-- Text Content -->
-				<h1>Subtitle goes here</h1>
-				<h2>Subtitle second line goes here lorem ipsum *</h2>
+			<div class="pricing-table__plans"> <!-- Plans content -->
+				<div class="pricing-table__plans-sub">
+					<h2>Subtitle goes here</h2>
+					<h3>Subtitle second line goes here lorem ipsum *</h3>
+				</div>
 				<ul>
 					<li>Drop out ramen hustle</li>
 					<li>Coworking viral landing</li>
 					<li>Crush revenue traction</li>
 					<li>User base minimum viable</li>
 					<li>Lorem ipsum dolor</li>
 				</ul>
+				<div class="pricing-table__buttons">
+					<a class="btn btn--blue" href="#">BUY TOUR</a>
+				</div>
 			</div>
+		</div> <!-- ***End of Pricing table 1*** -->
 
-			<div class="button"> <!-- Buy button -->
-				<form action="#">
-					<input class="button one" type="submit" name="btn_submit" value="BUY TOUR">
-				</form>
-			</div>
-		</div> <!-- ***End of Block 1*** -->
-
-		<div class="item"> <!-- ***Block 2 begins*** -->
-			<div class="image"> <!-- Image -->
+		<div class="pricing-table pricing-table--featured"> <!-- ***Pricing table 2 begins*** -->
+			<div class="pricing-table__bgimage"> <!-- Image -->
 				<img src="images/img2.jpg" alt="Mountains image">
-				<img src="images/glass.jpg" alt="Glass layer" class="glass">
-				<div class="new">
+				<img src="images/glass.jpg" alt="Glass layer" class="pricing-table__bgimage pricing-table__bgimage--blur">
+				<div class="pricing-table__sticker">
 					<img src="images/new.svg" alt="NEW">
-					<p>NEW</p>
+					<span class="pricing-table__sticker-text">NEW</span>
 				</div>
-				<div class="bigtext">
+				<span class="pricing-table__title">
 					MOUNTAINS
-					<p>The best price/quantity plan *</p>
-				</div>
+					<span class="pricing-table__title-sub">The best price/quantity plan *</span>
+				</span>
 			</div>
 
-			<div class="circle two"> <!-- Circle price tag -->
-				<p>$499</p>
-				<p>Per tour</p>
+			<div class="pricing-table__priceTag"> <!-- Circle price tag -->
+				<span class="pricing-table__price">$499</span>
+				<span class="pricing-table__price-sub">Per tour</span>
 			</div>
 
-			<div class="text"> <!-- Text Content -->
-				<h1>Subtitle goes here</h1>
-				<h2>Subtitle second line goes here lorem ipsum *</h2>
+			<div class="pricing-table__plans"> <!-- Plans content -->
+				<div class="pricing-table__plans-sub">
+					<h2>Subtitle goes here</h2>
+					<h3>Subtitle second line goes here lorem ipsum *</h3>
+				</div>
 				<ul>
 					<li>Drop out ramen hustle</li>
 					<li>Coworking viral landing</li>
 					<li>Crush revenue traction</li>
 					<li>User base minimum viable</li>
 					<li>Lorem ipsum dolor</li>
 				</ul>
-
-			</div>
-			<div class="button"> <!-- Buy button -->
-				<form action="#">
-					<input class="button two" type="submit" name="btn_submit" value="BUY TOUR">
-				</form>
+				<div class="pricing-table__buttons">
```

Comment:
có thể đặt tên như này
```suggestion
				<div class="pricing-table__button-wrapper">
```